### DISCUSSION TITLE
Navigation notification

### DISCUSSION THREAD

**Discussion by 13hannes11** [organicmaps/organicmaps#5302] [Ideas]
I think it would be useful to have a notification banner on android when navigation is active. This notification could display useful information but also allow an easier way back to the app.

Further this would allow my Bangle.js 2 watch to display the navigation instructions from the notification. I imagine other watches could benefit as well.

**Comment by biodranik** [comment]
Are there already related issues about background navigation?

**Reply by 13hannes11** [reply]
I could not find any but I find the github search not being the best.

**Comment by pm4rcin** [comment]
From my understanding the notification is also needed to allow the app to work in the background and to keep GPS lock while using other app.
